OpenTibiaUnity/AsyncGraphics.cs

```diff
@@ -51,8 +51,12 @@ public class AsyncGraphics
 
         public async Task DisposeOnDone(IEnumerable<Bitmap> bitmaps) {
             await Task.WhenAll(m_DrawTasks);
-            foreach (var bitmap in bitmaps)
-                bitmap.Dispose();
+            foreach (var bitmap in bitmaps) { 
+                if (bitmap != null) {
```

Comment:
what scenario would lead to a null bitmap?
i'm aware of that, so add a note about it